docs/docs/features/modules.mdx

```diff
@@ -109,6 +111,10 @@ manifest:
 </TabItem>
 </Tabs>
 
+### Building Locally
+
+To add a module to your build when building locally, you will need to clone/copy said module into your local file tree. You can then build using the module as described in ["Building with External Modules"](../development/build-flash.mdx#building-with-external-modules).
```

Comment:
```suggestion
To add a module to your build when building locally, you will need to clone/copy said module into your local file tree. You can then build using the module as described in [Building with External Modules](../development/build-flash.mdx#building-with-external-modules).
```
I think in general we aren't quoting section names? I had considered it but then thought hyperlink styling made it distinctive already.